app/models/katello/rhsm_fact_parser.rb

```diff
@@ -79,6 +79,22 @@ def boot_timestamp
       facts['proc_stat.btime']&.to_i
     end
 
+    def virtual
+      facts['virt.is_guest']
+    end
+
+    def ram
+      facts['memory.memtotal'] / 1024 if facts['memory.memtotal']
```

Comment:
Tested this and got the error `NoMethodError: undefined method '/' for "5944780":String
`.  Looks like the ram needs to be converted to an integer first.